tests/suites/test_suite_mpi.function

```diff
@@ -1,11 +1,26 @@
 /* BEGIN_HEADER */
 #include "mbedtls/bignum.h"
 #include "mbedtls/entropy.h"
+#include "bignum_core.h"
+#include "bignum_mod.h"
+#include "bignum_mod_raw.h"
+#include "constant_time_internal.h"
+#include "test/constant_flow.h"
 
 #if MBEDTLS_MPI_MAX_BITS > 792
 #define MPI_MAX_BITS_LARGER_THAN_792
 #endif
 
+#define ciL    (sizeof(mbedtls_mpi_uint))   /* chars in limb  */
+#define biL    (ciL << 3)                   /* bits  in limb  */
+
+/*
+ * Convert between bits/chars and number of limbs
+ * Divide first in order to avoid potential overflows
+ */
+#define BITS_TO_LIMBS(i)  ( (i) / biL + ( (i) % biL != 0 ) )
+#define CHARS_TO_LIMBS(i) ( (i) / ciL + ( (i) % ciL != 0 ) )
```

Comment:
(comment) the idiom I've always used is `( ( (i) + ciL - 1 ) / ciL )` but it appears Mbed TLS always does it this way (looking at `base64.c` and others)

Reply:
I'd never noticed this idiom, and always used (x+n-1)/n in the code I wrote.

Reply:
... and the BSDs typically have `#define howmany(x, y) (((x)+((y)-1))/(y))` in their `param.h`s, which I've also found convenient (along with `nitems()`)